package.json

```diff
@@ -35,6 +35,17 @@
     "test-all": "cross-env NODE_ENV=test yarn lint && yarn flow && yarn spec && yarn build",
     "version": "yarn build"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged && yarn test"
```

Comment:
I *strongly* suggest never using husky; autoinstalling git hooks is very hostile and dangerous, which is why git itself doesn't support it. Specifically, I've had a dependency's husky quietly install git hooks inside my employer's repo - it's a massive security risk.

Reply:
I had second thoughts about making this PR because of that very reason. Good call